.pre-commit-config.yaml

```diff
@@ -1,17 +1,21 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v4.4.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
     -   id: check-yaml
     -   id: check-added-large-files
 -   repo: https://github.com/pre-commit/mirrors-clang-format
-    rev: 'v14.0.0'
+    rev: v16.0.0
```

Comment:
This I'd keep 14, just because is the default version of Ubuntu 22.04, it might break with newer versions

Reply:
Alright, will revert it then.